README.md

```diff
@@ -76,42 +80,56 @@ node.run_state['splunk_ingredient'] = {
 }
 ```
 
-
 ### splunk_service
+
 Manages an installation of Splunk. Requires splunk_install to be evaluated first.
 
-**The following actions share the same properties:**
+**The following actions (start and restart) share the same properties**
+
+#### Action *:start*
 
-##### Action *:start*
 Starts the Splunk daemon if not already running.
 If the ulimit is changed, invokes a restart of the daemon at the end of the run.
 
-##### Action *:restart*
+#### Action *:restart*
+
 Restarts the Splunk daemon, or starts it if not already running.
 
 Properties:
 
 | Name    |               Type(s)               | Required | Default                                            | Description                                                                                                                                                                                      |
 |:--------|:-----------------------------------:|:--------:|:---------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| package | `:splunk` or `:universal_forwarder` | **Yes**  |                                                    | Specifies the Splunk package to install. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end` |
+| package | `:splunk` or `:universal_forwarder` | **Yes**  |                                                    | Specifies the installed Splunk package. You must specify the package, or name the resource for the package; for example, `package :splunk` or `splunk_install 'universal_forwarder' do ... end`  |
```

Comment:
minor, the example here could change to `splunk_service 'universal_forwarder' do ... end` instead of splunk_install.